python/doc/bibliography.rst

```diff
@@ -43,6 +43,13 @@ Bibliography
 .. [devroye1986] Devroye L, *Non-Uniform RandomVariate Generation*,
     Springer-Verlag, New York, 1986
     `pdf <http://doc.openturns.org/papers/devroye1986.pdf>`__
+.. [devroye1986b] Devroye L, *Non-Uniform RandomVariate Generation - Errata*,
```

Comment:
this one is not used, so we'll get an error:
`bibliography.rst:46:Citation [devroye1986b] is not referenced.`